arnoldPlugins/gaffer.mtd

```diff
@@ -1934,10 +1934,18 @@
 	gaffer.nodeMenu.category STRING "Light"
 	gaffer.graphEditorLayout.defaultVisibility BOOL false
 
+	gaffer.layout.activator.geometryIsPlane STRING "parameters['geometry_type'].getValue() == 'plane' and not parameters['shader'].getInput()"
+	gaffer.layout.activator.geometryIsBox STRING "parameters['geometry_type'].getValue() == 'box'"
```

Comment:
This isn't quite the same as the previous activator, which would also return `True` any time there wasn't a `shader` input. Was the old one wrong?

Reply:
I misread the original activator (which is correct), fixed and squashed into d28208f01f17b1ccc2c42386648f04b5bc9a1b8d